packages/mask/src/plugins/Game/Services/rss3.ts

```diff
@@ -0,0 +1,18 @@
+import { RSS3 } from '@masknet/web3-providers'
+import type { Constant } from '@masknet/web3-shared-base/src/utils/types'
+import { WalletRPC } from '../../Wallet/messages'
+import type { GameRSSNode } from '../types'
+import { GAME_CONFIG_ADDRESS } from '../constants'
+
+const cache = new Map<string, Record<string, Constant> | undefined>()
+
+export async function getConfigGameListFromRSS() {
+    const v = cache.get(GAME_CONFIG_ADDRESS)
+    if (v) return v
+    const rss = RSS3.createRSS3(GAME_CONFIG_ADDRESS, async (message: string) => {
+        return WalletRPC.signPersonalMessage(message, GAME_CONFIG_ADDRESS)
```

Comment:
Wallet RPC is refactoring to use internally. Please use the connection to sign a message.

Learn more: https://docs.mask.io/docs/web3/connection